lblod/app-digitaal-loket
# Adjusting form "Besluit over budget(wijziging) eredienstbestuur"

# Description

DL-4736

This PR adjusts the following form "Besluit over budget wijziging" which contains  a new input field : 

- Adds a new input field named "Inhoud Besluit" as conceptSchemeSelector with the codelist `http://lblod.data.gift/concept-schemes/7856206c-dfda-4163-8bd3-f465c794eced` 

This codelist is giving three options to choose from : Akteneming, Aanpassingsbesluit and Goedkeuringsbesluit

- Adding migrations to create new codelists

**Note : This PR needs to be merged with the form adjustement from "Links to other PR's" Section**

# Type of change


# Related services

- enrich-submission
- mu-migrations

# How to test 

1. Run this PR with https://github.com/lblod/enrich-submission-service/pull/15 change as enrich-submission image to load codelists (Simply build it and add it into your docker-compose.override.yml as service image)
2. Log in as Gemeente (any will do) and create a submission "Besluit over budget(wijziging) eredienstbestuur"

![Screenshot 2023-11-09 at 17-44-12 Loket voor lokale besturen](https://github.com/lblod/app-digitaal-loket/assets/38471754/d8feb185-8c01-4fb0-8524-e5d5190b0888)

3. Try to save it as concept 
4. Send it

This is what you should expect
![Screenshot 2023-11-10 at 15-02-08 Loket voor lokale besturen](https://github.com/lblod/app-digitaal-loket/assets/38471754/9452d01d-7c90-4da0-98f3-5ffb047d52ba)

5. Note this submission can be consumed by external applications like Toezicht-ABB

# What to check

- Any typo or semantic issues

# Links to other PR's

- Meldingsplichtige-api
- Toezicht-abb
- https://github.com/lblod/app-public-decisions-database/pull/20
- https://github.com/lblod/app-worship-decisions-database/pull/49
- https://github.com/lblod/manage-submission-form-tooling/pull/39
- https://github.com/lblod/enrich-submission-service/pull/15

# Notes

drc up -d enrich-submission; drc restart migrations resource cache
